Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyButton.cs

```diff
@@ -228,5 +243,11 @@ private void OnResourceRequested (object sender, EventArgs e)
 
 			resourceSelectorPopOver.Show (requestResourceView.Frame, (NSView)this, NSRectEdge.MinYEdge);
 		}
+
+		private void OnBindingRequested (object sender, EventArgs e)
+		{
+			var bindingEditorWindow = new BindingEditorWindow (this.hostResources, this.viewModel);
+			bindingEditorWindow.MakeKeyAndOrderFront (this);
```

Comment:
Should this be run modally too? There's nothing stopping someone from opening the binding editor, then clicking in the panel and opening a second one